extensions/ccui/layouts/UILayoutWebGLRenderCmd.js

```diff
@@ -130,16 +136,38 @@
     };
 
     proto._onBeforeVisitScissor = function(ctx){
-        var clippingRect = this._getClippingRect();
+        this._node._clippingRectDirty = true;
```

Comment:
This forces node to update clipping rect(this is another issue of not updating clipping rect after moving Layout)